### DISCUSSION TITLE
way to show schema and table for other project in big query

### DISCUSSION THREAD

**Discussion by jaythacker** [apache/superset#18305] [Q&A / Help]
My service account in big query has access to other project as well, and its required to join tables form both of the project.
I am able to write query and do join, but I am not able to see available tables in other project.
Is there any way to see that table in left side panel of SQL editor.

**Comment by betodealmeida** [comment]
Currently there's no way to do that, sorry. We need to do some refactoring to handle that — other DBs like Trino would also benefit from this.
